example-config.yml

```diff
@@ -15,6 +15,11 @@ api:
 #     lon: -122.71607145667079
 #     name: Oregon Zoo, Portland, OR
 
+### The default query parameters can be overridden be uncommenting this object.
+### Note: the override values must be valid values within otp-ui's query-params.js
```

Comment:
I think because these settings are drop-down items, the comment should rather say:

`... must be one of the values defined in otp-ui's query-params.js for the desired parameter`

Reply:
This is not *necessarily* true. There are some non-dropdown values not in the form that can be overridden (try `otherThanPreferredRoutesPenalty`). I think it's ok how it is for now.